### DISCUSSION TITLE
Images with spaces in file name not showing up in .md files

### DISCUSSION THREAD

**Discussion by jiachenyee** [community/community#58365] [Mobile]
### Select Topic Area

Bug

### Body

# Platform/Information
- GitHub iOS app (version 1.117.0)
- iOS 16.3.1/iOS 17

# Steps to reproduce
1. Create a repository and add an image file with a space in its name, for example, "my image.png"
2. Add that to a markdown file (ex: `![My Image](my%20image.png)`)
3. Open the Markdown file on the GitHub iOS app
4. The appears as a ?

# Sample Repo
- I have created a repository that recreates this issue.
- It has a README file with 2 images, one with a space, and the other without. 
- https://github.com/jiachenyee/test-repo. 

# Screenshot
<img src="https://github.com/community/community/assets/36725840/7164ae18-c90a-47ef-8e4a-9fe3739c6f6d" width="512px">

**Comment by Gavin-P-Smith** [comment]
hnh

**Reply by MindlessSteel** [reply]
Okay thanks for the info and now I am a tester with GitHub 
This should help me down the line using this protocol, software, etc because I am still a developer bye (DEFAULT)
🤔👀🙏🇺🇸

**Comment by iskae** [comment]
Hi @jiachenyee 👋 

Thanks for reaching out ❤️ We already have an internal tracking issue for this and working on a fix 🙇 I have also added your report in the tracking issue too 👍 

Please stay tuned for the future versions for the fix 🙇 

Cheers!
